tasks/update.py

```diff
@@ -0,0 +1,292 @@
+                skip_line_items[line_item['name']] = "attempt to target same video position multiple time"
+                continue
+
+            if targeted_positions_cnt > 1:
+                skip_line_items[line_item['name']] = "multiple video positions found, expecting only one position to update"
+                continue
+
+            # if the video position is not found, create a new targeting.videoPositionTargeting
+            if targeted_positions_cnt == 0:
+                line_item['targeting']['videoPositionTargeting'] = {"targetedPositions":[{"videoPosition":{"positionType": self.setting_class.NEW_VIDEO_POSITION}}]}
+
+            line_items_with_current_position[line_item['name']]=current_position
+            updated_line_items.append(line_item)
+
+        return updated_line_items, line_items_with_current_position, skip_line_items
+
+    def update(self):
+        """
+        Function to update "Video Position Targeting" of line items in GAM.
+        It retrieves line items from the response, selects line items to update, and then updates them using the LineItemService.
+        The user is prompted for confirmation before proceeding with the update.
+        If the update is successful, the names of the updated line items are logged.
+        """
+        response = self.get_line_items()
+        line_items = response['results'] if 'results' in response else []
+        if len(line_items) <= 0:
+            self.logger.info("No line item found for given input")
+            return
+
+        line_items_to_update, line_items_with_current_position, skip_line_items = self.select_line_items_to_update(line_items)
+        self.print_skipped_line_items(skip_line_items)
+        self.print_line_items_with_current_position(line_items_with_current_position)
+
+        if len(line_items_to_update) > 0:
+            user_confirmation = input("Do you want to proceed with the update? (y/n): ").lower()
+            if user_confirmation != 'y':
+                self.logger.info("Update canceled.")
+                return
+
+            line_item_service = self.ad_manager_client.GetService('LineItemService', version=self.API_VERSION)
+            returned_line_items=line_item_service.updateLineItems(line_items_to_update)
+            if len(returned_line_items) > 0:
+                self.logger.info("\nSuccessfully updated following line items")
+                for line_item in returned_line_items:
+                    self.logger.info(f"{self.color.BLUE}{line_item['name']}{self.color.END}")
+
+def main():
+    try:
+        logger = logging.getLogger(__name__)
+        if len(sys.argv) != 2:
+            logger.info("Usage: python -m task.update [VideoPosition|UpdatePrice]")
+            logger.info("Example: python -m task.update VideoPosition")
+            return
+
+        update_task = sys.argv[1]
+        color = Color()
+
+        # Use settings based on the command-line argument
+        if update_task == "VideoPosition":
```

Comment:
Can we make this case-insensitive, to prevent potential user errors?

Reply:
done